hack/cherry_pick_pull.sh

```diff
@@ -89,16 +95,38 @@ function return_to_kansas {
   if [[ "${gitamcleanup}" == "true" ]]; then
     git am --abort >/dev/null 2>&1 || true
   fi
-  git checkout -f "${STARTINGBRANCH}" >/dev/null 2>&1 || true
-  if [[ -n "${cleanbranch}" ]]; then
-    git branch -D "${cleanbranch}" >/dev/null 2>&1 || true
-  fi
-  if [[ -n "${prtext}" ]]; then
-    rm "${prtext}"
+
+  # return to the starting branch and delete the PR text file
+  if [[ -z "${DRY_RUN}" ]]; then
+    git checkout -f "${STARTINGBRANCH}" >/dev/null 2>&1 || true
+    if [[ -n "${cleanbranch}" ]]; then
+      git branch -D "${cleanbranch}" >/dev/null 2>&1 || true
```

Comment:
And if you don't delete the branch, maybe log its name, and the command to delete it?